static/js/factories/posts.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import R from 'ramda';
+
+import type { Post } from '../flow/discussionTypes';
+
+export const makePost = (isURLPost: boolean = false): Post => ({
```

Comment:
would probably be handy! I wrote a `randomString` function which is probably not the best thing ever, haha.